### PR TITLE
user/papers: update to 47.0

### PR DESCRIPTION
**Pull request by tulilirockz**

### CODE REVIEW

**Comment by tulilirockz** [comment]
i guess this should be merged with GNOME 47, right?

**Comment by EarldridgeJazzedPineda** [comment]
Congratulations on getting pull request number 3000!

You may want to consider putting this on hold, until the rest of the GNOME components have been update to version 47. Don't close this just yet.

**Comment by tulilirockz** [comment]
oh! cool! didnt even notice it! - oh yeah yeah, like, it already works fine -  doesnt seem to have any runtime errors - but still, probably should wait for the rest of the GNOME components yeah

File: user/papers/template.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "libxml2-progs",
     "meson",
     "pkgconf",
+    "rustfmt",
```
Comment (nekopsykose) [inline user/papers/template.py:21]: just patch this out by removing `cargo_fmt = find_program('cargo-fmt', required: true)`

**Comment by cassiofb-dev** [comment]
Congratz on pr #3000 @tulilirockz ! 🥳

**Comment by q66** [comment]
i missed this and updated it as a part of the overall batch :p sorry

**Comment by tulilirockz** [comment]
LOL its ok! just happy this got updated

**Pull request closed without merging**